frontend/server/src/main/java/org/pytorch/serve/wlm/BatchAggregator.java

```diff
@@ -46,7 +46,8 @@ public BaseModelRequest getRequest(String threadName, WorkerState state)
                 if (gpu != null) {
                     gpuId = Integer.parseInt(gpu);
                 }
-                return new ModelLoadModelRequest(model, gpuId);
+                String requestId = input.getRequestId();
```

Comment:
This request id is uuid (https://github.com/pytorch/serve/blob/master/frontend/server/src/main/java/org/pytorch/serve/wlm/WorkerThread.java#L331), not request id.

Reply:
Thanks, for pointing it out, I will get back with a proper fix. I am trying to search the relevant channel to get the request-id from. If you have some suggestion on where to look, please share.

Reply:
@lxning @maaquib Sorry, but I am having trouble getting the Request ID from client side. Can you give some hint?